### DISCUSSION TITLE
The personal access token you added to the .env file is invalid.

### DISCUSSION THREAD

**Discussion by fdfytr** [firefly-iii/firefly-iii#5030] [Firefly III] [answered]
I've successfully installed firefly and I'm trying to import YNAB via web server

Getting to second page for token confirmation and received above message
I had created a personal token at Firefly (which is 100s characters) and tried use that but it didnt work
I found .env file and used APPKEY ... I also tried command line token and same result

Would appreciate help on how to create tokens. I believe documentation page about how to create tokens will be useful

Thank you!

**Comment by JC5** [comment]
Does this page help?

https://docs.firefly-iii.org/other-data-importers/install/configure/#personal-access-token

**Reply by fdfytr** [reply]
Yes, it does! Thank you very much.

I went through document few times before but didnt thought 'Configuration' will have info about tokens... No idea why could not get it through search...

I actually dont know why it doesnt work then since I did exactly same as mentioned on that page... Will try again, may be bad copy&pasting

`Tried "http://172.19.0.3:8080/api/v1/about" 5 times but failed: cURL error 28: Connection timed out after 3142 milliseconds (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for http://172.19.0.3:8080/api/v1/about `

**Comment by fdfytr** [comment, marked as answer]
I found the problem
Docker containers for db and firefly app were on separate network from import tool - I had to change FIREFLY_URL to different ip and put importing tool on the same network.
